db/migrate/20200225220208_add_deactivated_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddDeactivatedToUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :deactivated, :boolean
```

Comment:
Let's ensure null is not possible here.

```suggestion
    add_column :users, :deactivated, :boolean, null: false, default: false
```